lacework-agent-windows/templates/windows-configmap.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.windowsAgent.enable -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: lacework-windows-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    tier: monitoring
+    app: {{ include "lacework-windows-agent.name" . }}
+    {{- with .Values.windowsAgent.agentConfig.labels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- with .Values.windowsAgent.agentConfig.annotations }}
+  annotations:
+    {{- toYaml . | nindent 8 }}
+  {{- end }}
+data:
+  config.json: |
+    {"autoupgrade": "{{ required "A value for autoupgrade is required" .Values.windowsAgent.agentConfig.autoUpgrade }}",
+     {{- with .Values.windowsAgent.agentConfig.fim }}
+     {{- if .enable }}
+     "fim": {
+             {{- if .fileIgnore }}"fileignore": {{ mustToJson .fileIgnore }}, {{- end }}
+             {{- if .filePath }}"filepath": {{ mustToJson .filePath }}, {{- end }}
+             "mode": "enable"
+             {{- if .noAtime }}, "noatime": "true" {{- end }}
+             {{- if .runAt }}, "runat": "{{ .runAt }}" {{- end }}
+             {{- if .maxscanfiles }}, "maxscanfiles": {{ .maxscanfiles }} {{- end }}
+     },
+     {{- else }}
+     "fim": {"mode": "disable"},
+     {{- end }}
+     {{- end }}
+  {{- if .Values.windowsAgent.agentConfig.proxyUrl }} "proxyurl" :"{{ .Values.windowsAgent.agentConfig.proxyUrl}}", {{- end }}
+  {{- if .Values.windowsAgent.agentConfig.registry }} "registry" :"{{ .Values.windowsAgent.agentConfig.registry}}", {{- end }}
+  {{- if .Values.windowsAgent.agentConfig.hids }} "hids": "{{ .Values.windowsAgent.agentConfig.hids}}", {{- end }}
+     "serverurl": "{{ .Values.windowsAgent.agentConfig.serverUrl}}",
+     "containers": {"mode": "enable"},
+     "tags": {"Env": "{{ .Values.windowsAgent.agentConfig.env }}"
+              {{- if .Values.windowsAgent.agentConfig.kubernetesCluster }}, "KubernetesCluster" :"{{ .Values.windowsAgent.agentConfig.kubernetesCluster}}" {{- end }}
+             }
+    }
+{{- end }}
```

Comment:
The whitespace formatting for this block is inconsistent. Not a blocker for getting this release out the door but you may want to fix this up so that it's nicely formatted when render.

You can easily check this by running `helm template -s templates/windows-configmap.yaml lacework-windows-agent`

Reply:
yes we should do that @hengw0498